website/cue/reference/components/sinks/clickhouse.cue

```diff
@@ -90,16 +90,56 @@ components: sinks: clickhouse: {
 			description: "The endpoint of the [Clickhouse](\(urls.clickhouse)) server."
 			required:    true
 			type: string: {
-				examples: ["http://localhost:8123"]
+				examples: ["http://localhost:8123", "tcp://localhost:9223"]
 			}
 		}
+		use_native_proto: {
+			description: "If true`, ClickHouse Native Protocol is used. Defaults to `false`, using `JSONEachRow` over HTTP."
+			required:    false
+			type: bool: default: false	
+		}
 		table: {
 			description: "The table that data will be inserted into."
 			required:    true
 			type: string: {
 				examples: ["mytable"]
 			}
 		}
+		sql_table_col_def: {
+			description: """
+				The clickhouse table column definition.
+				If use_native_proto is true, this field must be configured!
+				The key represents not only the column name of clickhouse table but also the key of the log.
+				The value now only supports limited type:
+				_type: UInt8,16,32,64  Int8,16,32,64  String FixedString(int) Float32,64 Date DateTime IPv4,6
+				Array(_type) Nullable(_type) Map(String, _type)
```

Comment:
Seems like no - this PR supports only a subset of Clickhouse types - https://github.com/suharev7/clickhouse-rs#supported-data-types . Some types are not supported yet (https://github.com/suharev7/clickhouse-rs/issues/178).

Since we cannot link even `clickhouse-rs` in this case, I suggest leaving it as is. I'll just reformat it a little bit.

Reply:
Fixed